tests/import/t21496.nim

```diff
@@ -0,0 +1,9 @@
+discard """
+  errormsg: "redefinition of 'm21496'; previous declaration here: t21496.nim(5, 12)"
+"""
+
+import fizz/m21496, buzz/m21496
```

Comment:
Ideally, this line would work on it's own, but any reference to `m21496` would cause an ambiguous error, no?